README.md

```diff
@@ -52,7 +52,7 @@ Example: California is UTC-08:00 (half the year) so I would make the TIMEZONE_OF
 - The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. If you choose not to use an alternate id, you can put the same column number in both CSV_COLUMN_NUMBER_FOR_GITHUB_ID and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID.
 
 ### To run:
-In your terminal, type `cat {path/to/CSVfile} | node index.js {date1} {date2}`  
+In your terminal, type `cat {path/to/CSVfile}.csv | node index.js {date1} {date2}`  
```

Comment:
I wonder if adding a separate example with .csv would be clearer. If we added .csv here, I think it would go inside the {} but am not certain.

Reply:
If that's the case, should we also change the example below where the .txt is outside the brackets as well?

Reply:
Yes, good catch. We should be consistent. If extensions outside the braces is a convention, I'm OK with having it that way both places. It's just not what I expected to see.

Reply:
Kevin and I chatted and we're going to leave it as-is